docs/kb/rules/ellipsis-metavariables.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Ellipsis metavariables can help with matching multiple word tokens.
+tags:
+  - Rules
+  - Semgrep Code
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Matching multiple tokens with ellipsis metavariables
+
+## What are ellipsis metavariables?
```

Comment:
```suggestion
## What are ellipsis metavariables?
```

I'd suggest avoiding hypophora in technical writing, whether body text or headings. We use two types of headings:
* Task-based
* Conceptual

Alternatives
* Introduction to ellipsis metavariables
* Defining ellipsis metavariables

Reply:
Thanks, I adjusted it!